addons/fleet_account/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+{
+    'name': 'Fleet - Account',
+    'version': '0.1',
```

Comment:
```suggestion
    'version': '1.0',
```
Nope ? :)

Reply:
I took the same as Fleet, but 1.0 is better